app/members/columns.tsx

```diff
@@ -1,5 +1,6 @@
+"use client";
```

Comment:
The reason why Columns.tsx is client-side is that the Data Table is also client-side. Data Table uses states, as it’s based on ShadCN. This means that we can’t simply make the Data Table server-side. Since Columns.tsx is used by the Data Table, it also can’t be made server-side. I would suggest leaving it as is unless it’s absolutely necessary for it to be server-side.